src/components/navMenu/mobileNavMenu.tsx

```diff
@@ -0,0 +1,101 @@
+import { dcNavLinks } from "@/data/dcNavLinks";
+import {
+  HStack,
+  Text,
+  Link,
+  Icon,
+  Button,
+  useCloseDropDown,
+} from "@nypl/design-system-react-components";
+import React, { useRef, useState } from "react";
+import FocusLock from "@chakra-ui/focus-lock";
+import { Box, chakra } from "@chakra-ui/react";
+import { useScrolled } from "@/hooks/useScrolled";
+
+const listItems = dcNavLinks.map(({ href, text }) => (
+  <Link
+    isUnderlined={false}
+    href={href}
+    key={text}
+    aria-label={text}
+    sx={{
+      display: "block",
+      fontSize: "s",
+      fontWeight: "510",
+      lineHeight: "150%",
+      paddingBottom: "m",
+      marginInlineStart: "xs",
+      width: "auto",
+      color: "ui.black",
+      ":hover": {
+        textDecoration: "none",
+      },
+      ":visited": {
+        color: "ui.black",
+      },
+    }}
+  >
+    {text}
+  </Link>
+));
+
+const MobileNavMenu = chakra(() => {
+  const [isOpen, setIsOpen] = useState<boolean>(false);
+  const ref = useRef<HTMLDivElement>(null);
+
+  useCloseDropDown(setIsOpen, ref);
+  const isScrolled = useScrolled("header", false);
+
+  return (
+    <Box ref={ref}>
+      <FocusLock isDisabled={!isOpen}>
+        <Button
+          aria-haspopup="true"
+          aria-label={isOpen ? "Close Navigation" : "Open Navigation"}
+          aria-expanded={isOpen ? true : null}
+          buttonType="text"
+          id="mobileNav-btn"
+          onClick={() => {
+            setIsOpen(!isOpen);
+          }}
+          sx={{
+            border: "none",
+            color: "ui.black",
+            ":hover": {
+              color: "ui.black",
+              bgColor: "ui.white",
+            },
+            "not:([disabled]):focus": {
+              outline: "none",
+            },
+          }}
+        >
+          <Icon name={isOpen ? "close" : "utilityHamburger"} size="large" />
+        </Button>
+        {isOpen && (
+          <HStack
+            id="mobile-menu"
+            sx={{
+              display: "flex",
+              flexDirection: "column",
+              alignItems: "left",
+              position: "fixed",
+              width: "100%",
+              paddingTop: "m",
+              paddingLeft: "xs",
+              marginTop: isScrolled ? "4px" : "-36px",
```

Comment:
Added an option to remove the delay from the scroll stickiness (`(useScrolled("header", false/true)` ) so that this could change immediately but even still there's a tiny but noticeable delay before the menu snaps into place under the header. Would love any ideas

Reply:
Also this shouldn't be hardcoded- I think the solution will probably have to do with `getBoundingClientRect()` but still working on how to incorporate that separately from `useScrolled`

Reply:
Is this question related to the gap that appears until you scroll past a certain point?
<img width="511" alt="Screen Shot 2023-12-06 at 3 58 25 PM" src="https://github.com/NYPL/digital-collections/assets/1280564/5663e77c-aba5-4317-87f6-cd081440a809">

Reply:
This worked for me but would like to know if this is how you expect it to work. Update position to absolute and keep marginTop to 4px.

```
position: "absolute",
width: "100%",
paddingTop: "m",
paddingLeft: "xs",
marginTop: "4px",
```

The margin is needed to keep it aligned and spaced and this works when there is no notification text or when there's long notification text as you scroll.

<img width="375" alt="Screen Shot 2023-12-06 at 8 22 25 PM" src="https://github.com/NYPL/digital-collections/assets/1280564/79d17896-e05d-4758-8722-7027f1781821">

Reply:
Oh man